clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -126,7 +126,7 @@ public static long readUnsignedInt(ByteBuffer buffer, int index) {
      * @return The integer read (MUST BE TREATED WITH SPECIAL CARE TO AVOID SIGNEDNESS)
      */
     public static int readUnsignedIntLE(InputStream in) throws IOException {
-        return (in.read() << 8 * 0)
+        return (in.read() & 0xff)
```

Comment:
I have been debating whether I should change this every time I saw it. :) I guess whoever wrote it liked the fact there was a regular pattern followed by each line.